CalibTracker/SiStripESProducers/plugins/fake/SiStripApvGainBuilderFromTag.cc

```diff
@@ -40,7 +40,7 @@ void SiStripApvGainBuilderFromTag::analyze(const edm::Event& evt, const edm::Eve
   inputApvGain.getDetIds(inputDetIds);
 
   // Prepare the new object
-  SiStripApvGain* obj = new SiStripApvGain();
+  SiStripApvGain obj = SiStripApvGain();
```

Comment:
This syntax is actually calling the copy constructor I think.  SiStripApvGain obj; should be enough.